### PR TITLE
Feature/vendor fixes

### PR DESCRIPTION
**Pull request by turegjorup**

### CODE REVIEW

File: config/packages/framework.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ framework:
 
         # Redis
         app: cache.adapter.redis
-        default_redis_provider: redis://localhost/1
+        default_redis_provider: redis://redis:6379/1
```
Comment (cableman) [inline config/packages/framework.yaml]: 👍

File: src/Service/VendorService/AbstractBaseVendorService.php [hunk 2]
```diff
@@ -164,60 +166,94 @@ protected function updateOrInsertMaterials(array &$identifierImageUrlArray, stri
...
+     * @param array $updatedIdentifiers
+     *   Updated identifiers
+     * @param array $insertedIdentifiers
+     *   Inserted identifiers
+     * @param string $identifierType
+     *   The type of identifiers in to be processed
+     */
+    private function sendCoverImportEvents(array $updatedIdentifiers, array $insertedIdentifiers, string $identifierType): void
+    {
+        if ($this->queue) {
```
Comment (cableman) [inline src/Service/VendorService/AbstractBaseVendorService.php]: Why this test?

File: src/Service/VendorService/BogPortalen/BogPortalenVendorService.php [hunk 3]
```diff
@@ -254,15 +252,22 @@ private function listZipContents($path): array
...
     {
         $isbnList = [];
 
         foreach ($fileNames as $fileName) {
-            $isbnList[] = substr($fileName, -17, 13);
+            $isbn = substr($fileName, -17, 13);
+
+            // Ensure that the found string is a number to filter out
+            // files with wrong or incomplete isbn numbers1
+
```
Comment (cableman) [inline src/Service/VendorService/BogPortalen/BogPortalenVendorService.php]: Why the empty line after this comment.... its hanging

File: src/Service/VendorService/BogPortalen/BogPortalenVendorService.php [hunk 4]
```diff
@@ -254,15 +252,22 @@ private function listZipContents($path): array
...
     private function getIsbnNumbers(array &$fileNames): array
     {
         $isbnList = [];
 
         foreach ($fileNames as $fileName) {
-            $isbnList[] = substr($fileName, -17, 13);
+            $isbn = substr($fileName, -17, 13);
+
+            // Ensure that the found string is a number to filter out
+            // files with wrong or incomplete isbn numbers1
```
Comment (cableman) [inline src/Service/VendorService/BogPortalen/BogPortalenVendorService.php]: numbers1? => numbers.

File: src/Service/VendorService/AbstractBaseVendorService.php (hunk 2 above)
Comment (turegjorup) [reply src/Service/VendorService/AbstractBaseVendorService.php]: We have a `queue` option for the vendor command that can be set to false if you don't want the imprt to go to the queue

Comment (cableman) [reply src/Service/VendorService/AbstractBaseVendorService.php]: 👍

**Pull request merged**